CHANGES.rst

```diff
@@ -66,6 +66,8 @@ Other Changes and Additions
 - Redshifts imported with a custom line list are now ignored.  Redshift must be set app-wide via 
   viz.set_redshift or the line list plugin. [#1134]
 
+- Subset selection dropdowns in plugins now show synced color indicators. [#1156]
```

Comment:
Do you think this is more of a "new feature"?

Reply:
I don't have a strong feeling either way, it just feels like too small of a change (for the user) to qualify as a "feature", but have no problem moving it if you think it belongs there?